docker-compose.yaml

```diff
@@ -104,14 +103,17 @@ services:
     networks:
       - docker_example_network
 
-
   graphdb:
-    image: ontotext/graphdb:10.6.4
+    image: khaller/graphdb-free
```

Comment:
Not sure how we are deploying graphdb right now, guess this is just something we can keep a mental note of/call out for Andrew since I wonder if this image will be maintained long term.